src/main/webapp/style.css

```diff
@@ -0,0 +1,98 @@
+#content {
```

Comment:
Should we add license here as well?
Verify and accordingly take a call.

Reply:
Also, it will be nice if we have similar CSS across the project.
(Low priority)